jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/NewActionViewImpl.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.cm.client.actions;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.jboss.errai.common.client.dom.HTMLElement;
+import org.jboss.errai.common.client.dom.MouseEvent;
+import org.jboss.errai.common.client.dom.Span;
+import org.jboss.errai.common.client.dom.TextInput;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.EventHandler;
+import org.jboss.errai.ui.shared.api.annotations.ForEvent;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+
+import org.jbpm.workbench.cm.client.util.FormGroup;
+import org.jbpm.workbench.cm.client.util.FormLabel;
+import org.jbpm.workbench.cm.client.util.Modal;
+import org.jbpm.workbench.cm.client.util.ValidationState;
+import org.uberfire.mvp.Command;
+
+import static com.google.common.base.Strings.*;
+import static org.jbpm.workbench.cm.client.resources.i18n.Constants.*;
+
+@Dependent
+@Templated
+public class NewActionViewImpl implements CaseActionsPresenter.NewActionView {
+
+    @Inject
+    @DataField("action-name-group")
+    FormGroup actionNameGroup;
+
+    @Inject
+    @DataField("action-name-label")
+    FormLabel actionNameLabel;
+
+    @Inject
+    @DataField("action-name-input")
+    TextInput actionNameInput;
+
+    @Inject
+    @DataField("action-name-help")
+    Span actionNameHelp;
+
+    @Inject
+    @DataField("action-desc-input")
+    TextInput actionDescInput;
+
+    @Inject
+    @DataField("action-users-group")
+    FormGroup usersGroup;
+
+    @Inject
+    @DataField("action-users-input")
+    TextInput usersInput;
+
+    @Inject
+    @DataField("action-groups-group")
+    FormGroup groupsGroup;
+
+    @Inject
+    @DataField("action-groups-input")
+    TextInput groupsInput;
+
+    @Inject
+    @DataField("assignation-help")
+    Span assignationHelp;
+
+
+    @Inject
+    @DataField("modal")
+    private Modal modal;
+
+    private Command okCommand;
+
+    @Inject
+    private TranslationService translationService;
+
+    @PostConstruct
+    public void init() {
+
```

Comment:
please review formatting , there are a few empty lines that could be removed across different files.